src/lib/formatters/sql.py

```diff
@@ -1,11 +1,16 @@
 from pandas import DataFrame
 
+from src.lib.sql import Sql
+
 
 class SQLFormatter(object):
     """Class that receive pandas dataframe
     and write it down in .sql format
     """
     key = 'sql'
+    modes = {'append': 'append',
+             'truncate': 'replace',
```

Comment:
It's correct ?

Reply:
SIm, esta correto. Esta propriedade modes é utilizada no modulo __to_db, onde é escrita no banco de dados via pandas. E o pandas possui apenas dois modos de escrita, append(adicionando linhas) e replace(tirando tudo e criando novamente)

Reply:
Hmm, thank you for explication.